lib/manageiq/cross_repo/runner/github.rb

```diff
@@ -12,18 +13,21 @@ def self.available?
 
       private
 
-      def travis_config
+      def ci_config
+        github_config = YAML.load_file(CONFIG_FILE)
+
         steps = github_config["jobs"]["ci"]["steps"]
+        steps_by_name = steps.index_by { |step| step["name"] }
+
         language = steps.any? { |s| s["uses"] == "ruby/setup-ruby@v1" } ? "ruby" : "node_js"
```

Comment:
This might need both...ui classic has both node and ruby, for example.

Reply:
So this is actually for manageiq-ui-service where the language in the .travis.yml was node_js, manageiq-ui-classic still had ruby as the language: https://github.com/ManageIQ/manageiq-ui-classic/pull/8098/files#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485L2

The classic UI repo installs node conditionally based on the TEST_SUITE

Reply:
> The classic UI repo installs node conditionally based on the TEST_SUITE

Not anymore :(

Reply:
Thinking we should just, by default, install both Ruby and Node in the base manageiq-cross-repo workflow.  I always thought it weird that ui classic intalls node itself on demand.

Reply:
Okay that is going to take some more refactoring that I was planning on doing in a phase 2, we have defaults per language and can only have one language right now.  Could we cross-repo just the ruby test_suite on ui-classic for now?

Reply:
So we can't run `setup-node` without the repo having a `yarn.lock` file, and there is no `none` option for `cache` like there is with bundler

Reply:
Think you figured this out already, but you can just _not_ put the `cache` option at all. With setup-ruby, if you leave out the bundler-cache entirely it defaults to, I think "Gemfile.lock or bundle lock" (which is "true"), and that's why they have a "none"